README.md

```diff
@@ -1 +1,50 @@
-# rawhide-rpmdeplint-pipeline
+# rpmdeplint-pipeline
+
+This repository contains definition of the rpmdeplint CI pipeline.
+
+This pipeline runs [rpmdeplint](https://github.com/fedora-ci/rpmdeplint-image) tool on RPM packages and publishes results via [CI Messages](https://pagure.io/fedora-ci/messages).
+
+> ** **Important:** ** Structure of the branches in this repository resembles braches in dist-git, i.e. master branch contains the pipeline definition for Fedora Rawhide, [f32](https://github.com/fedora-ci/rpmdeplint-pipeline/tree/f32) branch is for Fedora 32, etc.
+
+## Test definition
+
+The generic rpmdeplint test is described in a [Flexible Metadata Format](https://pagure.io/fedora-ci/metadata).
+
+The actual definition lives in the [rpmdeplint.fmf](./rpmdeplint.fmf) file.
+
+## Test execution
+
+The pipeline delegates the test execution to the Testing Farm (TODO: link once available). It only collects, archive and report results.
```

Comment:
Here, I would clarify the "it" as either the pipeline or testing farm; I assume it's the pipeline.

Reply:
Good catch, updated. Thanks ;)